README.md

```diff
@@ -12,18 +12,59 @@ Copy `config.js.template` to `config.js`, and then fill in the variables. Finall
 
 You can enable a simple API Key verification in the `config.js` file. In this case, your requests must include the header `X-API-KEY` with the correct key.
 
-If you are using cURL to test, you can include the header using the `-H` parameter, e.g., `curl -H "X-API-Key: YourKey" http://localhost:8000/0/balance`.
+If you are using cURL to test, you can include the header using the `-H` parameter, e.g., `curl -H "X-API-Key: YourKey" http://localhost:8000/wallet/balance`.
 
 It follows the [Swagger Specification for API Keys](https://swagger.io/docs/specification/authentication/api-keys/).
 
 ## How to use?
 
 Check out the full documentation in the OpenAPI Documentation in `api-docs.js`.
 
+### Start a wallet
+
+Create and start a new wallet.
+
+Parameters:
+
+`key`: key reference of this wallet in the object of created wallets. Used to say which wallet each request is directed to.
```

Comment:
I'd say: a unique identifier for the new wallet. It will be required to access this wallet later.

Reply:
> unique identifier for the new wallet

It sounds like `wallet-id` describes this better than `key` (which I think is too generic). If we have to name this parameter I'd prefer something like `wallet-id`. I like the idea of using headers, which would need a name for the parameter. Some alternatives are `wallet-index`/`wallet-idx` (which to me sound more like something that would be a small integer: `wallet-idx=0`, `wallet-idx=1`, ...)

An alternative would be arranging this so it doesn't need a name (like a positional argument). Maybe in the URL.

Reply:
Done!